test/patient.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { browser } from 'protractor';
+import { LandingPage } from '../src/page/landing.page';
+import { PatientDataPage } from '../src/page/patient-data.page';
+import { ConfirmationPage } from '../src/page/confirmation.page';
+
+describe('Add a new patient', () => {
```

Comment:
It is better if you follow gherking. For example:
Given I want to add a patient
When I fill all the data required
And I clic on save button
Then I expect a confirmation message

Reply:
agree